rpm/0001-Preserve-process-effective-group-for-privileged-grou.patch

```diff
@@ -1,14 +1,14 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Andrew Branson <andrew.branson@jolla.com>
-Date: Wed, 15 Apr 2020 14:12:17 +0200
+Date:  Wed, 15 Apr 2020 14:12:17 +0200
```

Comment:
These extra spaces on some Date: lines, but not on all of the patches, look strange. This really git format-patch output here?

Reply:
This was because I used the approach of:
1) Check out v0.9.66
2) Apply patches one by one
3) Rebase v0.9.72 on top
4) Run git format-patch

This resulted in patches that were correct, but had the wrong creator (=me) and the wrong timestamp (=the time when I had applied the patch) so I manually corrected these. My editor (unbeknownst to me) tried to be clever and "helpfully" hid any superfluous spaces, which is why I didn't notice them. This should now be fixed.

Reply:
Did you use "git am" for applying the patches?

Reply:
Trying to remember... I may have used "git patch" and that's the reason.

Reply:
Alright. Just to note that the git tooling does support having the patches applied with the original authors. If not done this time, something to remember for the next one.